fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -686,7 +722,7 @@ def initialize(user = nil, password = nil, use_shell_script = false, provider_sh
       use_shell_script ||= Helper.windows?
       use_shell_script ||= Feature.enabled?('FASTLANE_ITUNES_TRANSPORTER_USE_SHELL_SCRIPT')
 
-      if jwt.to_s.empty?
+      if jwt.to_s.empty? && api_key.nil?
```

Comment:
can anyone think of any reason to continue asking the user for input when the api key is set?